services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -1460,6 +1483,24 @@ public Builder excludeList(@Nullable List<String> exclude) {
       return this;
     }
 
+    /**
+     * Exclude certain road types or points from routing.
+     * The default is to not exclude anything from the profile selected.
+     *
+     * @param exclude an object of excludes that are used during the request.
+     *   Use {@link Exclude.Builder} to build exclude.
+     * @return this builder for chaining options together
+    */
+    @NonNull
+    public Builder excludeObject(@Nullable Exclude exclude) {
```

Comment:
Why is this `@Nullable`?

Reply:
To let users cleanup the exclude. 
```kotlin
  @Test
  public void cleanExcludes() {
    RouteOptions routeOptions = routeOptions().toBuilder()
        .excludeObject(null)
        .build();

    assertNull(routeOptions.exclude());
  }
```

Reply:
What if you build new route options from existing route options as on example ☝️